clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -129,7 +138,18 @@ public void test_capabilities_exposed() throws Exception {
     assertThat(capabilities[3].getName(), equalTo("SettingsCapability"));
     assertThat(capabilities[4].getName(), equalTo("ManagementAgentService"));
     assertThat(capabilities[0].getDescriptors(), hasSize(4));
-    assertThat(capabilities[1].getDescriptors(), hasSize(75));
+
+    Collection<Descriptor> descriptors = capabilities[1].getDescriptors();
+    Collection<Descriptor> allDescriptors = new ArrayList<Descriptor>();
+    allDescriptors.addAll(StatsUtil.CACHE_DESCRIPTORS);
+    allDescriptors.addAll(StatsUtil.ONHEAP_DESCRIPTORS);
+    allDescriptors.addAll(StatsUtil.OFFHEAP_DESCRIPTORS);
+    allDescriptors.addAll(StatsUtil.CLUSTERED_DESCRIPTORS);
+
+    assertThat(descriptors, hasSize(allDescriptors.size()));
+    assertThat(descriptors, containsInAnyOrder(allDescriptors.toArray()));
```

Comment:
please make this assertion before the hasSize. Otherwise the error message we'll get will still be 50 vs 55 and the list of missing stats